main.js

```diff
@@ -55,27 +73,26 @@
         });
     }
 
-    //Displays and hides the sidebar for mobile
-    function displaySidebar() {
-      document.getElementById("sidebar").style = "display:block";
-      document.getElementById("mobileMenuIcon").style = "display:none";
-      document.getElementById("mobileHideMenu").style = "display:block";
-    }
-
-    function hideSidebar() {
-      document.getElementById("sidebar").style = "";
-      document.getElementById("mobileMenuIcon").style = "";
-      document.getElementById("mobileHideMenu").style = ""
+    //Displays and hides the sidebar
+    function toggleSidebar() {
+      if (sidebarToggle.open == 1) {
+          sidebarToggle.open = 0;
+          sidebar.style.display = "none";
```

Comment:
again, maybe just use a class name for the display. cuts thes LOCs in the process, too.